lldb/include/lldb/API/SBDebugger.h

```diff
@@ -321,13 +321,22 @@ class LLDB_API SBDebugger {
 
   void SetLoggingCallback(lldb::LogOutputCallback log_callback, void *baton);
 
+  /// DEPRECATED: We used to only support one Destroy callback. Now that we
+  /// support Add and Remove, you should only remove Destroy callbacks that
+  /// you Add-ed. Use Add and Remove instead.
+  ///
+  /// Clear all previously added callbacks and only add the given one.
   void SetDestroyCallback(lldb::SBDebuggerDestroyCallback destroy_callback,
```

Comment:
There are some macros you can use in addition to the documentation: `LLDB_DEPRECATED_FIXME`. This will give compiler warnings to anyone using `SetDestroyCallback` so they can update their code.

Reply:
Thanks for the tip! Will find an example and use it.

Reply:
Added.  I assume I only need to add it to SBDebugger.h, not Debugger.h.  Hope the message/fix I wrote is also good.